boards/pba-d-01-kw2x/include/periph_conf.h

```diff
@@ -97,7 +97,7 @@ static const uart_conf_t uart_config[] = {
         .dev    = UART0,
         .clken  = (volatile uint32_t*)(BITBAND_REGADDR(SIM->SCGC4, SIM_SCGC4_UART0_SHIFT)),
         .freq   = CLOCK_CORECLOCK,
-        .pin_rx = GPIO_PIN(PORT_D, 6),
```

Comment:
I don't have this board so I can't test this, could you open a separate PR for this, please?